src/components/pages/Friends.tsx

```diff
@@ -1,23 +1,84 @@
+import SyncIcon from '@mui/icons-material/Sync'
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
 import Container from '@mui/material/Container'
+import FormControlLabel from '@mui/material/FormControlLabel'
+import FormGroup from '@mui/material/FormGroup'
 import Stack from '@mui/material/Stack'
+import Switch from '@mui/material/Switch'
+import Typography from '@mui/material/Typography'
+import { useContext, useEffect, useState } from 'react'
 import { Link } from 'react-router-dom'
+import useFriends from '../../hooks/useFriends'
+import Spinner from '../partials/Spinner'
+import { UserContext } from '../partials/UserContext'
 
 export default function Friends() {
-  // TODO: replace with 'Friends' objects from useUser()
-  const friends = ['Jake', 'Jason', 'Jamie']
+  const { currentUser } = useContext(UserContext)
+  const username = currentUser!.username
+  const [enabled, setEnabled] = useState<boolean>(false)
+  const {
+    friends,
+    loading: loadingFriends,
+    error: errorFetchingFriends,
+  } = useFriends({ username: username })
+
+  useEffect(() => {
+    if (friends.length !== 0) {
```

Comment:
how do we identify if the user has no friend vs not enabled yet?

Reply:
hmm didn't think abut that edge case. when we discussed, we assumed empty [] would mean it is not enabled yet. Should we add a boolean field in the model for isEnabled?